packages/renderer/src/lib/network/CreateNetwork.svelte

```diff
@@ -0,0 +1,321 @@
+          <ContainerConnectionDropdown
+            id="providerChoice"
+            name="providerChoice"
+            bind:value={networkInfo.selectedProvider}
+            connections={providerConnections} />
+        </div>
+      {/if}
+
+      <div hidden={createNetworkInProgress}>
+        <label for="subnet" class="block mb-2 font-semibold text-[var(--pd-content-card-header-text)]">Subnet</label>
+        <Input
+          bind:value={networkInfo.subnet}
+          name="subnet"
+          id="subnet"
+          placeholder="Subnet (e.g., 10.89.0.0/24 or 2001:db8::/32)"
+          class="w-full" />
+      </div>
+
+      <div hidden={createNetworkInProgress}>
+        <label for="ipv6" class="block mb-2 font-semibold text-[var(--pd-content-card-header-text)]"
+          >IPv6 (Dual Stack)</label>
+        <SlideToggle id="ipv6" name="ipv6" bind:checked={networkInfo.ipv6Enabled} />
+      </div>
+
+      <div hidden={createNetworkInProgress}>
+        <label for="ipRange" class="block mb-2 font-semibold text-[var(--pd-content-card-header-text)]">IP range</label>
+        <Input
+          bind:value={networkInfo.ipRange}
+          name="ipRange"
+          id="ipRange"
+          placeholder="IP range (e.g., 10.89.0.0/25 or 2001:db8::/64)"
+          class="w-full" />
+      </div>
+
+      <div hidden={createNetworkInProgress}>
+        <label for="gateway" class="block mb-2 font-semibold text-[var(--pd-content-card-header-text)]">Gateway</label>
+        <Input
+          bind:value={networkInfo.gateway}
+          name="gateway"
+          id="gateway"
+          placeholder="Gateway (e.g., 10.89.0.1 or 2001:db8::1)"
+          class="w-full" />
+      </div>
+
+      <div hidden={createNetworkInProgress}>
+        <label for="internal" class="block mb-2 font-semibold text-[var(--pd-content-card-header-text)]"
+          >Internal</label>
+        <SlideToggle id="internal" name="internal" bind:checked={networkInfo.internalEnabled} />
+      </div>
+
+        <div hidden={createNetworkInProgress}>
+        <label for="driver" class="block mb-2 font-semibold text-[var(--pd-content-card-header-text)]">Driver</label>
+        <Dropdown
+          id="driver"
+          name="driver"
+          bind:value={networkInfo.driver}
+          options={availableDrivers}
+          class="w-full" />
+      </div>
+
+      <div hidden={createNetworkInProgress}>
+        <label
+          for="options"
+          class="block mb-2 font-semibold text-[var(--pd-content-card-header-text)]"
+          >Advanced Options (driver-specific):</label
+        >
+      
+        {#each networkInfo.options as _, index (index)}
+          <div class="flex flex-col w-full py-1">
+            <div class="flex flex-row justify-center items-center w-full">
+              <Input
+                bind:value={networkInfo.options[index]}
+                placeholder="e.g., com.docker.network.bridge.name=mybr0"
+                class="ml-2 w-full" />
```

Comment:
```suggestion
                class="w-full" />
```

<img width="273" height="218" alt="Image" src="https://github.com/user-attachments/assets/4329e0fd-175d-4973-b462-a91b2bca412f" />